openfasoc/generators/inductor-generator/tools/ind_specs.yaml

```diff
@@ -0,0 +1 @@
+gen_name: "Inductor Generator"
```

Comment:
Where is the rest of the config file? Should at least include a basic example inductor config to show the proper structuring of the config file.